packages/ra-ui-materialui/src/input/AutocompleteInput.spec.tsx

```diff
@@ -694,60 +650,6 @@ describe('<AutocompleteInput />', () => {
         });
     });
 
-    it('should not render a LinearProgress if loading is true and a second has not passed yet', () => {
```

Comment:
Mui autocomplete has its own loading mechanism. Maybe we should check we don't its loading prop unless a second has passed

Reply:
I see that you did it, am I wrong?